install_executorch.py

```diff
@@ -226,6 +235,9 @@ def main(args):
             "-m",
             "pip",
             "install",
+        ]
+        + (["--editable"] if args.editable else [])
```

Comment:
Thoughts on making this default? Or at least have a PR in draft mode to test the whole CI pipeline with editable?